src/me/actions/thunks/index.ts

```diff
@@ -44,7 +45,7 @@ export const getMe = () => async dispatch => {
 export const getQuartzMe = () => async dispatch => {
   try {
     dispatch(setQuartzMeStatus(RemoteDataState.Loading))
-    const resp = await apiGetQuartzMe()
+    const resp = await getMeQuartzGen({})
```

Comment:
why is there an empty object passed in here?

Reply:
TS issues. Based on the yml definition there's an expectation that a parameter will be passed in, but that parameter is blank

Reply:
does it make sense to update the yaml definition so that this does not require sending an empty object?

Reply:
Would if I could. I tried making that happen but it doesn't look like the parameters are ever set in the `yml` definition. I think it might be the result of the `oats` conversion of the `yml` file.  For reference, here's the definition:


```
'/me':
    get:
      operationId: GetMe
      tags:
        - Account
        - Operator
        - Region
        - Organization
      responses:
        '200':
          description: congregated meta data for an individual account and org
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Me'
        '401':
          description: Unauthorized
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Error'
        '404':
          description: Cannot get account or organization for user
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Error'
        default:
          description: Unexpected error
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Error'
                ````

Reply:
oh yea that doesn't do anything with params. bummer that thats needed for the code.

Reply:
Yup. Definitely tech debt with our dependence on `oats` as an SDK generator.